sentry-android-core/src/main/java/io/sentry/android/core/AppStartState.java

```diff
@@ -44,7 +44,7 @@ void setAppStartEnd(final long appStartEndMillis) {
   }
 
   @Nullable
-  synchronized Long getAppStartInterval() {
+  public synchronized Long getAppStartInterval() {
```

Comment:
this needs to be public to be used in hybrid SDKs, e.g.
reason is https://github.com/getsentry/sentry-java/blob/main/sentry-android-core/src/main/java/io/sentry/android/core/PerformanceAndroidEventProcessor.java#L63-L64